AuthenticatorShared/UI/Vault/ItemList/ItemList/ItemListProcessorTests.swift

```diff
@@ -501,9 +501,14 @@ class ItemListProcessorTests: AuthenticatorTestCase { // swiftlint:disable:this
             ItemListItem.fixture(name: "Beta"),
             ItemListItem.fixture(name: "Delta"),
             ItemListItem.fixture(name: "Alpha"),
+            ItemListItem.fixture(name: "beta"),
         ]
         let resultSection = ItemListSection(id: "", items: results, name: "")
-        let resultSorted = ItemListSection(id: "", items: results.sorted(by: { $0.name < $1.name }), name: "")
+        let resultSorted = ItemListSection(
+            id: "",
+            items: results.sorted { $0.name.localizedStandardCompare($1.name) == .orderedAscending },
```

Comment:
🤔 I know previously we did it algorithmically, but I think there could be value in explicitly stating what we expect the  order of the items to be. Not enough to not approve this, but just something I've been thinking about.

Reply:
I'm cool with that. I had the exact same thought while I was doing this - i.e. if it had been manually assigned, maybe the test would have caught it originally. I kinda leaned towards keeping it algorithmic in the moment, but hearing you say that too pushes me back to manually specifying it. I'll throw that in a commit. 👍

Reply:
Definitely plays into the whole "testing behavior" vs "testing implementation" thing 🤔

Reply:
Updated in latest. 👍